### PR TITLE
add polling to gamepad

### PR DESCRIPTION
**Pull request by noah-nm**
One less thing in the constructor!

### CODE REVIEW

File: src/menus/MainMenu.java [hunk 1]
```diff
@@ -25,16 +24,16 @@ public void play() {
...
+                gamepad.poll();
             }
 
             this.dc.clear();
 
             // background
             this.dc.setOrigin(DConsole.ORIGIN_TOP_LEFT);
             this.dc.setPaint(new Color(121, 224, 218, 200));
-            this.dc.fillRect(0, 0, this.dc.getWidth(), this.dc.getHeight());
+            this.dc.fillRect(0, 0, 1200, this.dc.getHeight());
```
Comment (HoodieRocks) [inline src/menus/MainMenu.java]: you forgot that you changed this

**Review by HoodieRocks — DISMISSED** [review]
LGTM

File: src/menus/MainMenu.java (hunk 1 above)
Comment (noah-nm) [reply src/menus/MainMenu.java]: oopsie

**Review by HoodieRocks — APPROVED** [review]
LGTM

**Pull request merged**